src/tox_lsr/test_scripts/runqemu.py

```diff
@@ -920,10 +920,6 @@ def run_ansible_playbooks(  # noqa: C901
 def install_requirements(sourcedir, collection_path, test_env, collection):
     """
```

Comment:
```suggestion
    """Install reqs from {meta,tests}/collection-requirements.yml, if any."""
```
This should fix the pydocstyle error.